colour/algebra/tests/tests_random.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Defines unit tests for :mod:`colour.algebra.random` module.
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+import sys
+
+if sys.version_info[:2] <= (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from colour.algebra import random_triplet_generator
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2014 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Production'
+
+__all__ = ['RANDOM_TRIPLETS',
+           'TestRandomTripletGenerator']
+
+RANDOM_TRIPLETS = np.array([
+    [0.96702984, 0.54723225, 0.97268436],
+    [0.71481599, 0.69772882, 0.2160895],
+    [0.97627445, 0.00623026, 0.25298236],
+    [0.43479153, 0.77938292, 0.19768507],
+    [0.86299324, 0.98340068, 0.16384224],
+    [0.59733394, 0.0089861, 0.38657128],
+    [0.04416006, 0.95665297, 0.43614665],
+    [0.94897731, 0.78630599, 0.8662893],
+    [0.17316542, 0.07494859, 0.60074272],
+    [0.16797218, 0.73338017, 0.40844386]])
+
+
+class TestRandomTripletGenerator(unittest.TestCase):
+    """
+    Defines :func:`colour.algebra.random.random_triplet_generator` definition
+    unit tests methods.
+    """
+
+    def test_random_triplet_generator(self):
+        """
+        Tests :func:`colour.algebra.random.random_triplet_generator`
+        definition.
+        """
+
+        prng = np.random.RandomState(4)
```

Comment:
I don't think the expected behavior (consistency of the random numbers) here is guaranteed across systems/versions. [This SO question](http://stackoverflow.com/questions/8786084/reproducibility-of-python-pseudo-random-numbers-across-systems-and-versions?rq=1) seem to confirm this too.
I guess as long as it doesn't make problems this is not a big issue, but using a mock random number generator might be the "more correct" solution here to avoid this from breaking in the future.